src/entities/update-entities.js

```diff
@@ -62,8 +63,8 @@ const filterSSLData = response => {
     // Depending on the length of the certificate chain, there may be multiple owner fields. We want the last one, which is most likely
     // to be the name of the actual company.
     // TODO check whois data for alt subjects on cert, add to entity if data matches
-    let sslNames = response.match(/O = .*,/g) || []
-    let sslAltNames = response.match(/DNS:.*/g) || []
+    const sslNames = response.match(/O = .*,/g) || []
+    // const sslAltNames = response.match(/DNS:.*/g) || []
```

Comment:
@dharb  also not used - not sure if a bug or just code that was left out

Reply:
Yeah remnants of more code that I removed. When I originally built the entity list I pulled in domains listed under SSL alt names. It was great for building out breadth for each entity, but it unfortunately produced a lot of false positives that I had to filter out (due to shared SSL certs).

I decided to remove it from the update script and currently use a different script that requires a manual check before anything is added to our data. I'd like to revisit at some point and get that in open source as well, but just haven't had a chance.